### PR TITLE
Remove XISF wrapper and adjust changes for LibXISF 0.2.0

### PR DESCRIPTION
**Pull request by nouspiro**

### CODE REVIEW

File: libs/indibase/CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 3.13)
 project(indidriver C CXX)
 
+set(CMAKE_CXX_STANDARD 17)
+
```
Comment (knro) [inline libs/indibase/CMakeLists.txt:5]: This should be specified in cmake_modules/CMakeCommon.cmake line 14

**Comment by knro** [comment]
Ok looks like std::random_shuffle needs to be replaced as well by std::shuffle

**Comment by nouspiro** [comment]
Hm just found out that std::from_chars is not implemented on older gcc versions. Gotta replace it with something else.

**Comment by nouspiro** [comment]
Wanted to use std::from_chars and std::to_chars to get locale independed string->float conversion but it is not implemented properly. I see that indi use AutoCNumeric to ensure correct conversion so I will use that.

**Pull request merged**